pyproject.toml

```diff
@@ -7,7 +7,13 @@ license = "MIT"
 readme = "README.md"
 repository = "https://github.com/meltano/meltano"
 classifiers = [
-  "Programming Language :: Python :: 3",
+  "Development Status :: 5 - Production/Stable",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.7",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
```

Comment:
Some of of these seem to be added automatically: 
<img width="248" alt="Screenshot 2023-01-30 at 20 04 16" src="https://user-images.githubusercontent.com/16805946/215641150-f6872d27-c06b-4eab-bc3d-80066c0b6d0b.png">

It's probably ok to be explicit, though

Reply:
Oh that's strange - I added them explicitly because I recall seeing them missing from Meltano on PyPI. Perhaps I was looking at a different view, or maybe I just misremembered.